clusters/app.ci/ipi-deprovision/ipi-deprovision-bc.yaml

```diff
@@ -62,3 +62,17 @@ items:
       type: Docker
     triggers:
     - type: ConfigChange
+    - type: ImageChange
+      imageChange: {}
```

Comment:
huh is the empty one needed too?

Reply:
/approve
/lgtm
/hold

Feel free to unhold if yes

Reply:
Empty one is for the CentOS base image.

/hold cancel